speakeasy/winenv/api/usermode/kernel32.py

```diff
@@ -3286,6 +3286,8 @@ def CreateFile(self, emu, argv, ctx={}):
         if ad:
             argv[1] = ' | '.join(ad)
 
+        disp_bytes = disp.to_bytes(8, 'little')
+        disp = int(int.from_bytes(disp_bytes[0:4], 'little')
```

Comment:
Looks like we are missing a parentheses here.

Reply:
Whoops :) Should be fixed now.